subsys/net/l2/wifi/wifi_mgmt.c

```diff
@@ -902,6 +902,27 @@ static int wifi_set_enterprise_creds(uint32_t mgmt_request, struct net_if *iface
 NET_MGMT_REGISTER_REQUEST_HANDLER(NET_REQUEST_WIFI_ENTERPRISE_CREDS, wifi_set_enterprise_creds);
 #endif
 
+static int wifi_set_bss_max_idle_period(uint32_t mgmt_request, struct net_if *iface,
+				  void *data, size_t len)
+{
+	const struct device *dev = net_if_get_device(iface);
+	const struct wifi_mgmt_ops *const wifi_mgmt_api = get_wifi_api(iface);
+	unsigned short *bss_max_idle_period = data;
+
+	if (wifi_mgmt_api == NULL || wifi_mgmt_api->set_bss_max_idle_period == NULL) {
+		return -ENOTSUP;
+	}
+
+	if (!data || len != sizeof(*bss_max_idle_period)) {
+		return -EINVAL;
+	}
+
+	return wifi_mgmt_api->set_bss_max_idle_period(dev, *bss_max_idle_period);
```

Comment:
In PR https://github.com/zephyrproject-rtos/hostap/pull/30 the only Zephyr interface is `wpa_cli` and here we are using net_mgmt which one are the drivers supposed to use?